python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -1047,6 +1047,26 @@ def aten〇index〇Tensor(self: List[int], indices: List[Optional[List[int]]]) -
 def aten〇cat(tensors: List[List[int]], dim: int = 0) -> List[int]:
     return upstream_shape_functions.cat(tensors, dim)
 
+def aten〇split(self: List[int], split_sizes: List[int], dim: int = 0) -> List[List[int]]:
+    return upstream_shape_helpers.split(self, split_sizes, dim)
+
+def aten〇split〇sizes(self: List[int], split_size: List[int], dim: int = 0) -> List[List[int]]:
+    return upstream_shape_helpers.split(self, split_size, dim)
+
+def aten〇split_with_sizes(self: List[int], split_sizes: List[int], dim: int = 0) -> List[List[int]]:
+    return upstream_shape_helpers.split(self, split_sizes, dim)
+
+def aten〇split〇Tensor(self: List[int], split_size: int, dim: int = 0) -> List[List[int]]:
+    dim_size = self[dim]
+    chunks = (dim_size + split_size - 1) // split_size
+    split_sizes = [split_size for i in range(chunks)]
+    split_sizes[chunks - 1] = split_size - (split_size * chunks - dim_size)
+    return upstream_shape_helpers.split(self, split_sizes, dim)
```

Comment:
These shape functions are outdated. The module `upstream_shape_helpers` should no longer be used in torch-mlir. See: https://github.com/llvm/torch-mlir/pull/1019. The shape functions should be upstreamed into pytorch, and in this file you should use the corresponding `upstream_shape_functions.` function.

I think this is causing the tests to crash. When I try running a simple lowering I get the following error:

```
-:6600:10: error: 'func.call' op '__torch__.torch_mlir.dialects.torch.importer.jit_ir.build_tools.upstream_shape_helpers.maybe_wrap_dim' does not reference a valid function
-:6600:10: note: see current operation: %4 = "func.call"(%arg2, %3, %2) {callee = @__torch__.torch_mlir.dialects.torch.importer.jit_ir.build_tools.upstream_shape_helpers.maybe_wrap_dim} : (!torch.int, !torch.int, !torch.bool) -> !torch.int
```